stable/fluentd/templates/hpa.yaml

```
@@ -7,13 +7,12 @@ metadata:
     app: {{ template "fluentd.name" . }}
     chart: {{ template "fluentd.chart" . }}
     release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
+    heritage: {{ .Release.Service }}    
```

```suggestion
    heritage: {{ .Release.Service }}
```

Remove the tab at the end please